### DISCUSSION TITLE
Which statusline plugin is used for the pictures of Readme ?

### DISCUSSION THREAD

**Discussion by jqhr** [rmehri01/onenord.nvim#32] [Q&A] [answered]
I think that statusline is very cool.And I want it's config too

**Comment by rmehri01** [comment, marked as answer]
Thanks, I'm using the [config from LunarVim](https://github.com/LunarVim/LunarVim/tree/rolling/lua/lvim/core/lualine), which uses [lualine](https://github.com/nvim-lualine/lualine.nvim)!

**Reply by jqhr** [reply]
great!
